.github/workflows/run_batch_script.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       with:
         repository: spinalcordtoolbox/spinalcordtoolbox
         path: spinalcordtoolbox
+        ref: jn/4805-fix-output-suffix-of-contrast-agnostic-model  # Temporary branch until PR is merged
```

Comment:
The PR is failing due to https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4805. I've got a fix on the above branch that I've added in <pr goes here>. When that PR gets merged, we can remove this `ref:` field and test against `master` again.